go.mod

```diff
@@ -151,7 +151,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/gohai v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/obfuscate v0.59.0
 	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.59.0
-	github.com/DataDog/datadog-agent/pkg/security/secl v0.56.0
```

Comment:
Do we really need that ? I think there is a replace anyway

Reply:
:+1: